.github/workflows/rust.yml

```diff
@@ -103,6 +116,19 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Generate Cargo.lock
+        run: cargo +nightly update
```

Comment:
Why use nightly here?

Reply:
I use `CARGO_UNSTABLE_SPARSE_REGISTRY: "true"` to speedup fetching of crates.io
It cuts the fetching time from 30s to 1s, eliminating the need of git cloning.